hw/ip/otbn/dv/otbnsim/sim/state.py

```diff
@@ -110,9 +133,19 @@ def commit(self, sim_stalled: bool) -> None:
         # If error bits are set, pending_halt should have been set as well.
         assert self._err_bits == 0
 
-        if self._new_rnd_data:
-            self.wsrs.RND.set_unsigned(self._new_rnd_data)
-            self._new_rnd_data = None
+        # RTL must send a RND done signal 5 cycles after calculation
+        assert self.rnd_cdc_counter
```

Comment:
Does this depend on the relationship between the two clocks? Maybe we should make this a bit laxer ("At most 10 cycles" or something?)